From: James McCoy <jamessan@debian.org>
Date: Wed, 1 Aug 2018 20:44:36 -0400
Subject: svnshell

Bug #234462: The error messages that svnshell spits out for unknown
Bug #234462: The error messages that svnshell spits out for unknown
commands are confusing. They should be replaced by a "Unknown command."
error message.  Upstream says we have no sense of humor.  (:

Patch from Roland Illig <roland.illig@gmx.de>.
---
 tools/examples/svnshell.py | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/tools/examples/svnshell.py b/tools/examples/svnshell.py
index 9c67af4..b05e7ef 100755
--- a/tools/examples/svnshell.py
+++ b/tools/examples/svnshell.py
@@ -56,14 +56,8 @@ class SVNShell(Cmd):
   def postcmd(self, stop, line):
     self._setup_prompt()
 
-  _errors = ["Huh?",
-             "Whatchoo talkin' 'bout, Willis?",
-             "Say what?",
-             "Nope.  Not gonna do it.",
-             "Ehh...I don't think so, chief."]
-
   def default(self, line):
-    print(self._errors[randint(0, len(self._errors) - 1)])
+    print "Unknown command."
 
   def do_cat(self, arg):
     """dump the contents of a file"""
